includes/functions/llms.functions.templates.dashboard.php

```diff
@@ -193,7 +193,7 @@ function lifterlms_template_my_courses_loop( $student = null, $preview = false )
 
 		if ( ! $courses['results'] ) {
 
-			printf( '<p>%s</p>', __( 'You are not enrolled in any courses.', 'lifterlms' ) );
+			printf( '<p>%s</p>', apply_filters( 'lifterlms_not_enrolled_text', __( 'You are not enrolled in any courses.', 'lifterlms' ) ) );
```

Comment:
```suggestion
			/**
			 * Not enrolled text.
			 *
			 * Allows developers to filter the text to be displayed when the student is not enrolled in any courses.
			 *
			 * @since [version]
			 *
			 * @param string $not_enrolled_text The text to be displayed when the student is not enrolled in any course.
			 */
			printf( '<p>%s</p>', apply_filters( 'lifterlms_not_enrolled_text', __( 'You are not enrolled in any courses.', 'lifterlms' ) ) );
```

Reply:
Thanks, Lee! I've added the changelog and the docblock.

Reply:
Even better folks if we do it this way:
```php
			printf( 
				'<p>%s</p>', 
				/**
				 * Not enrolled text.
				 *
				 * Allows developers to filter the text to be displayed when the student is not enrolled in any courses.
				 *
				 * @since [version]
				 *
				 * @param string $not_enrolled_text The text to be displayed when the student is not enrolled in any course.
				 */
				apply_filters( 'lifterlms_not_enrolled_text', esc_html__( 'You are not enrolled in any courses.', 'lifterlms' ) ) 
			);
```

oh any way that allows us to have the filter docblock just above the filter itself.
Alternatively we can even create a variable and then `printf` the variable. You see what I mean?

Reply:
Thanks, @seothemes @eri-trabiccolo for the suggestions.

I think we can avoid making a variable for now as it's only used in one instance in the function. Just updated the code with the suggestions.